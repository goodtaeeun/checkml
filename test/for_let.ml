let for_example () =
  for i = 1 to 5 do
    Printf.printf "i = %d\n" i
  done
