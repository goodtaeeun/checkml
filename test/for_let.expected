File for_let.ml, line 2, column 2
[31mError[0m: Don't use for!

